index.js

```diff
@@ -0,0 +1,66 @@
+document.addEventListener('DOMContentLoaded', () => {
+    getFilm()
+
+    let review = document.getElementById("review");
+    let list = document.getElementById("list");
+    let form = document.querySelector("form");
+
+    form.addEventListener("submit", (event) => {
+        event.preventDefault();
+        let li = document.createElement("li")
+        li.innerText = review.value
+        list.appendChild(li);
+
+    })
+
+})
+
+
+
+
+const getFilm = () => {
+    const select = document.getElementById('select')
+    const option = document.createElement('option')
+    option.classList.add("option")
+    option.appendChild(document.createTextNode('Text'));
+    option.value = 'value';
+    select.appendChild(option);
+
+    const url = "https://ghibliapi.herokuapp.com/films"
+    fetch(url)
+        .then(response => response.json())
+        .then(films => {
+
+            for (let i = 0; i < films.length; i++) {
+                let film = films[i].title
+                option.innerHTML = films[0].title
+                displayMovieInfo(films[i])
+
+            }
+
+        })
+}
+
+
+
+const displayMovieInfo = (films) => {
```

Comment:
Because this function is called inside for loop, you are creating movie info for 20 movies, instead of one movie that a user selects from options.